.semaphore/playwright-e2e.yml

```diff
@@ -77,6 +77,133 @@ blocks:
+              $token = $(vault write -field=token "auth/semaphore_self_hosted/login" role="default" jwt="$Env:SEMAPHORE_OIDC_TOKEN")
+              vault login -no-print token=$token
+            }
+          # Check if Docker is available for @local tests
+          - |
+            Write-Output "Checking Docker availability..."
+            try {
+              $dockerVersion = docker --version 2>$null
+              if ($LASTEXITCODE -eq 0) {
+                Write-Output "Docker is available: $dockerVersion"
+                make load-cached-docker-images
+              } else {
+                Write-Output "Docker is not available. @local tests will be skipped."
+              }
+            } catch {
+              Write-Output "Docker is not available. @local tests will be skipped."
+            }
+          # Windows version of make setup-test-env
+          - |
+            $E2E_USERNAME = $(vault kv get -field=E2E_USERNAME v1/ci/kv/vscodeextension/testing)
+            Set-Content -Path .env -Value "E2E_USERNAME=$E2E_USERNAME"
+            $E2E_PASSWORD = $(vault kv get -field=E2E_PASSWORD v1/ci/kv/vscodeextension/testing)
+            Add-Content -Path .env -Value "E2E_PASSWORD=$E2E_PASSWORD"
+            $E2E_KAFKA_API_KEY = $(vault kv get -field=E2E_KAFKA_API_KEY v1/ci/kv/vscodeextension/testing)
+            Add-Content -Path .env -Value "E2E_KAFKA_API_KEY=$E2E_KAFKA_API_KEY"
+            $E2E_KAFKA_API_SECRET = $(vault kv get -field=E2E_KAFKA_API_SECRET v1/ci/kv/vscodeextension/testing)
+            Add-Content -Path .env -Value "E2E_KAFKA_API_SECRET=$E2E_KAFKA_API_SECRET"
+            $E2E_SR_API_KEY = $(vault kv get -field=E2E_SR_API_KEY v1/ci/kv/vscodeextension/testing)
+            Add-Content -Path .env -Value "E2E_SR_API_KEY=$E2E_SR_API_KEY"
+            $E2E_SR_API_SECRET = $(vault kv get -field=E2E_SR_API_SECRET v1/ci/kv/vscodeextension/testing)
+            Add-Content -Path .env -Value "E2E_SR_API_SECRET=$E2E_SR_API_SECRET"
+            $E2E_SR_URL = $(vault kv get -field=E2E_SR_URL v1/ci/kv/vscodeextension/testing)
+            Add-Content -Path .env -Value "E2E_SR_URL=$E2E_SR_URL"
+            $E2E_KAFKA_BOOTSTRAP_SERVERS = $(vault kv get -field=E2E_KAFKA_BOOTSTRAP_SERVERS v1/ci/kv/vscodeextension/testing)
+            Add-Content -Path .env -Value "E2E_KAFKA_BOOTSTRAP_SERVERS=$E2E_KAFKA_BOOTSTRAP_SERVERS"
+      jobs:
+        - name: "Playwright: E2E Tests (VS Code ${{parameters.VSCODE_VERSION}})"
+          commands:
+            - npm ci --prefer-offline --include=dev
+            - npx playwright install
+            # Windows version of test-playwright-e2e, but exclude @local tests until Docker is available
+            - |
+              if ("$Env:TEST_SUITE_TAG" -ne "" -and "$Env:TEST_SUITE_TAG" -ne "TEST_SUITE_TAG") {
+                npx gulp e2e -t "$Env:TEST_SUITE_TAG" -x "@local"
+              } else {
+                npx gulp e2e -x "@local"
```

Comment:
We _could_ set up some new `TEST_SUITE_EXCLUDE_TAG` env var support here, but we really only need to exclude `@local` tests on the Windows agent in CI. If we need to start excluding other test patterns in CI, we can revisit env var support that can be used more flexibly with Semaphore task params.